tensorflow/workspace.bzl

```diff
@@ -164,8 +164,15 @@ def tf_repositories(path_prefix = "", tf_repo_name = ""):
     tf_http_archive(
         name = "mkl_dnn",
         build_file = clean_dep("//third_party/mkl_dnn:mkldnn.BUILD"),
+<<<<<<< HEAD
```

Comment:
This seems to have conflict markers left it. Probably a failed merge conflict resolution?

Reply:
Thanks, fixed it.